### PR TITLE
Merge all Public Cloud docs in one file

### PR DESCRIPTION
**Pull request by ldevulder**

### CODE REVIEW

File: src/installation/install_epinio_on_public_cloud.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<summary>Microsoft AKS configuration</summary>
+
+### AKS prerequisites
+
+* Epinio has been tested with AKS version **v1.20.9** and **v1.21.7**
+* To just try out Epinio, e.g. 2 **Standard_D2_v2** nodes are sufficient
+
+### Create an AKS cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) to create an AKS cluster.
```
Comment (juadk) [inline src/installation/install_epinio_on_public_cloud.md:17]:
```suggestion
If you do not have an existing cluster, follow the [quickstart](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough) to create an AKS cluster.
```

File: src/installation/install_epinio_on_public_cloud.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<summary>Amazon EKS configuration</summary>
+
+### EKS prerequisites
+
+* Epinio has been tested with EKS version **v1.20.7** and **v1.21.2**
+* To just try out Epinio, e.g. 2 **t3a.large** nodes are sufficient
+
+### Create an EKS cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) to create an EKS cluster.
```
Comment (juadk) [inline src/installation/install_epinio_on_public_cloud.md:30]:
```suggestion
If you do not have an existing cluster, follow the [quickstart](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) to create an EKS cluster.
```

File: src/installation/install_epinio_on_public_cloud.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+<summary>Google GKE configuration</summary>
+
+### GKE prerequisites
+
+* Epinio has been tested with GKE version **v1.20.7**, **v1.21.2** and **v1.21.5**
+* To just try out Epinio, e.g. 1 **n2-standard-4** node is sufficient
+
+### Create a GKE cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart) to create a GKE cluster.
```
Comment (juadk) [inline src/installation/install_epinio_on_public_cloud.md:43]:
```suggestion
If you do not have an existing cluster, follow the [quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart) to create a GKE cluster.
```

File: src/installation/install_epinio_on_rancher_desktop.md [hunk 4]
```diff
@@ -30,13 +30,13 @@ export EPINIO_SYSTEM_DOMAIN=${LB_IP}.omg.howdoi.website
 ```
 Previous commands are for Bash, but can be easily adapted for Windows shell.
 
-If you don't want to use the default "magic" DNS please follow [Install Epinio with "magic" DNS](./install_epinio_magicDNS.md).
+If you don't want to use the default "magic" DNS please follow [Install Epinio with custom DNS](install_epinio_customDNS.md).
```
Comment (juadk) [inline src/installation/install_epinio_on_rancher_desktop.md:33]:
```suggestion
If you do not want to use the default "magic" DNS please follow [Install Epinio with custom DNS](install_epinio_customDNS.md).
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Noticed an issue in inherited text which was not changed, and, IMHO, is in need of an update. Best do it now.

**Review by juadk — CHANGES_REQUESTED** [review]

File: src/installation/system_requirements.md [hunk 5]
```diff
@@ -5,10 +5,10 @@ For the Epinio server, and related deployments we recommend to consider the foll
...
   - Linux
-    - k3s, k3d
-    - minikube (not yet)
+    - K3s, K3d
+    - minikube
   - Windows
-    - [Rancher Desktop](https://rancherdesktop.io)
+    - [Rancher Desktop](install_epinio_on_rancher_desktop.md)
   - MacOS
 - Kubernetes versions 1.18 .. 1.21
```
Comment (juadk) [inline src/installation/system_requirements.md:13]: This is not the scope of this PR but I think we need to update this too:
Feel free to commit it or I can do it in another PR.
```suggestion
- Kubernetes versions 1.20 .. 1.23
```

File: src/installation/install_epinio_on_public_cloud.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+</details>
+
+## Installation on Public Clouds
+
+Beside advanced installation options, there are two ways of installing Epinio:
+
+1. [Installation using a MagicDNS Service](./install_epinio_magicDNS.md)
+
+- For test environments, this should work on nearly any kubernetes distribution. Epinio will try to automatically create a magic DNS domain, e.g. **10.0.0.1.omg.howdoi.website**.
```
[the remark this answers is not in the document]
Comment (ldevulder) [reply src/installation/install_epinio_on_public_cloud.md:52]: Sure, I will change this.

**Review by juadk — APPROVED** [review]

File: src/installation/install_epinio_on_public_cloud.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ldevulder) [reply src/installation/install_epinio_on_public_cloud.md:52]: In fact yes the "magic" DNS cannot be used on Public Clouds tests, so better to remove it.
And I will add the command to explain how to get the IP address, it's the one provided at the end of the helm installation in fact.

[the remark this answers is not in the document]
Comment (ldevulder) [reply src/installation/install_epinio_on_public_cloud.md]: For K3d there is already the needed information to get the IP address, will add it for K3s.

[the remark this answers is not in the document]
Comment (ldevulder) [reply src/installation/install_epinio_on_public_cloud.md]: Should be better now, let me know :-)

**Review by andreas-kupries — APPROVED** [review]
LGTM

File: src/installation/install_epinio_on_public_cloud.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply src/installation/install_epinio_on_public_cloud.md]: Looks good. Approved.

**Pull request merged**